<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/PublishIcon1.png</normaloff>../icons/PublishIcon1.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_heading">
     <property name="font">
      <font>
       <family>Punishment</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Studio Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layout_project" stretch="0,1,0,0">
     <item>
      <widget class="QLabel" name="label_project_folder_linux">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Project folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="settings_tab_linux">
        <attribute name="title">
         <string>Linux</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="folder_select_layout">
           <item>
            <widget class="QLineEdit" name="lineEdit_project_linux"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_open_project_linux">
             <property name="text">
              <string>Open</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="settings_tab_win">
        <attribute name="title">
         <string>Windows</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="folder_select_layout_3">
           <item>
            <widget class="QLineEdit" name="lineEdit_project_win"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_open_project_win">
             <property name="text">
              <string>Open</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="settings_tab_mac">
        <attribute name="title">
         <string>Mac OS</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="folder_select_layout_6">
           <item>
            <widget class="QLineEdit" name="lineEdit_project_mac"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_open_project_mac">
             <property name="text">
              <string>Open</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_projec_prefix">
       <property name="text">
        <string>Prefix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_project_prefix">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layout_render" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_render_folder">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Render folder</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="folder_select_layout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_render"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layout_squence" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sequence Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="folder_select_layout_4">
       <item>
        <widget class="QLineEdit" name="lineEdit_sequence"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layout_shot" stretch="0,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Shot Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="folder_select_layout_5">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_shot"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../fonts/Punishment-DPwE.ttf"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
